druid-shell/src/platform/mac/window.rs

```diff
@@ -191,7 +220,8 @@ impl WindowBuilder {
                 NSSize::new(self.size.width, self.size.height),
             );
 
-            let window = NSWindow::alloc(nil).initWithContentRect_styleMask_backing_defer_(
+            let window_a: id = msg_send![WINDOW_CLASS.0, alloc];
```

Comment:
I found "window_a" a confusing name.

Reply:
I changed it to just rebind window... I was going for 'window that has been allocated but not initialised' as doing the alloc and the init in one line looks pretty confusing with the msg_send macro.